models/.gitignore

```diff
@@ -0,0 +1,2 @@
+/misogyny_tfidf.joblib
```

Comment:
I'm not sure if it's a very good idea to have many different `.gitignore` files. It's usually easier to track all of them in one file in the root folder.